android/src/main/java/com/concurrentsound/ConcurrentSoundModule.kt

```diff
@@ -115,14 +181,23 @@ class ConcurrentSoundModule internal constructor(context: ReactApplicationContex
     playerToUse.setVolume(to, to)
   }
 
-  @RequiresApi(Build.VERSION_CODES.M)
   @ReactMethod
   fun setPlaybackRate(key: String?, uri: String?, to: Float, promise: Promise) {
```

Comment:
do we need `override` here?

Reply:
i also did now see `abstract function` in `android/src/oldarch/ConcurrentSoundSpec.kt`

Reply:
No. Since this is not a built in method in MediaPlayer.
Adding `override` gave error.

Reply:
yes, you should add `abstract function` in android/src/oldarch/ConcurrentSoundSpec.kt

Reply:
after that have `override ` in `android/src/main/java/com/concurrentsound/ConcurrentSoundModule.kt`

Reply:
this one will help to support both old and new arch

Reply:
i updated it by myself